packages/common/src/types.ts

```diff
@@ -153,8 +153,16 @@ export type Plugin<T = Schema> = {
   uninterruptedEditMode?: boolean;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type Plugins = { [key: string]: Plugin<any> | undefined };
+export type Plugins = { [key: string]: Plugin<any> };
```

Comment:
I couldn't see a reason why an element of a the plugins object could be `undefined`?